src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/SyntaxTokenClassifier.cs

```diff
@@ -44,7 +44,9 @@ internal class SyntaxTokenClassifier : AbstractSyntaxClassifier
                     var types = semanticModel.LookupTypeRegardlessOfArity(identifier, cancellationToken);
                     if (types.Any(s_shouldInclude))
                     {
+#nullable disable
```

Comment:
Ditto: why?

Reply:
The reported warning is `GetClassificationForType` can return null, but `ClassificationSpan` does not allow a null type. The resolution would likely be one of the following:

1. Demonstrate that `GetClassificationForType` does not return null in this scenario (and add an assertion)
2. Update this method to avoid adding a classification for null and/or default to using Text if the call returned null
3. Update `ClassificationSpan` to allow null in the constructor, either by automatically substituting Text or by preserving null as an allowed value

Reply:
I'd do 2, since it seems like there's a legitimate bug otherwise here.

Reply:
(or 1, but 3 seems not right).

Reply:
In either case, it seems odd that when we find an actual bug we then `#nullable disable`.... :smile:

Reply:
Updated to document the reason for current exclusion. It can be cleaned up during a smaller, localized follow-up pass.